example/src/pages/credential/Credential.css

```diff
@@ -0,0 +1,17 @@
+.conatiner {

```

Comment:
Is this only needed if the author likes to change things around?  By default, this shouldn't be needed, correct?

Reply:
Yes it's based on the user need